src/quo2/components/community/token_gating.cljs

```diff
@@ -0,0 +1,168 @@
+(ns quo2.components.community.token-gating
+  (:require [quo.react-native :as rn]
+            [quo2.components.avatars.channel-avatar :as channel-avatar]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.icon :as icon]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.tags.token-tag :as token-tag]
+            [quo2.foundations.colors :as colors]
+            [quo.theme :as theme]
+            [status-im.i18n.i18n :as i18n]))
+
+(def styles {:container {:flex 1}
+             :header-container {:flex-direction :row
+
+                                :align-items :center}
+             :header-spacing-community {:margin-bottom 16}
+             :header-spacing-channel {:margin-bottom 24}
```

Comment:
Please align map key-values ([ref](https://github.com/bbatsov/clojure-style-guide#map-keys-alignment))

Reply:
Should all map values be at the same column as well?

Reply:
It appears, `make lint-fix` and Calva Formatting doesn't fix this automatically...

Reply:
> Should all map values be at the same column as well?

I am not sure what you mean. do you mean nested maps? Its up to you, but the main point is to make the code more readable, for example

![image](https://user-images.githubusercontent.com/17097240/193913647-64b88cff-4ca5-433a-a24e-99689ee1ba7d.png)